src/Api/AdminConsole/Models/Response/Organizations/OrganizationIntegrationResponseModel.cs

```diff
@@ -21,4 +19,29 @@ public OrganizationIntegrationResponseModel(OrganizationIntegration organization
     public Guid Id { get; set; }
     public IntegrationType Type { get; set; }
     public string? Configuration { get; set; }
+
+    public OrganizationIntegrationStatus Status => Type switch
+    {
+        // Not yet implemented, shouldn't be present, leave Initiated for now
+        IntegrationType.CloudBillingSync => OrganizationIntegrationStatus.Initiated,
+        IntegrationType.Scim => OrganizationIntegrationStatus.Initiated,
```

Comment:
🎨 With these out of scope really I'd let them fall into a `default` case, and that should probably be assumed as `Completed`. Or, do we return a status of "Invalid" for both this and a default?